src/test/java/org/jenkinsci/plugins/ewm/ConfigAsCodeTest.java

```diff
@@ -0,0 +1,27 @@
+package org.jenkinsci.plugins.ewm;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.Matchers.is;
+import org.jvnet.hudson.test.JenkinsRule;
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+
+/**
+ * Test for Configuration As Code Compatibility.
+ *
+ * @author Martin d'Anjou
+ */
+public class ConfigAsCodeTest {
+
+    @ClassRule public static JenkinsRule r = new JenkinsRule();
+
+    @Test
+    public void should_support_configuration_as_code() throws Exception {
+        String config = ConfigAsCodeTest.class.getResource("configuration-as-code.yaml").toString(); //configuration-as-code.yml").toString();
+        System.out.println("config: " + config);
+        ConfigurationAsCode.get().configure(config);
+        assertTrue(true); // check plugin has been configured as expected );
```

Comment:
this should have an assertion that shows the plugin is configured as expected...

Reply:
Oh yeah this code is not finished yet (far from complete ...) and I will add test code when the dummy test passes. Thank you !